src/lib/pgp-key.cpp

```diff
@@ -1830,8 +1869,16 @@ pgp_key_t::is_signer(const pgp_subsig_t &sig) const
         return sig.sig.keyfp() == fp();
     }
     if (!sig.sig.has_keyid()) {
-        return false;
+        return false || (
+#if defined(ENABLE_CRYPTO_REFRESH)
```

Comment:
@TJ-91 I currently don't see the code removed. In any case, maybe you can elaborate a bit on which packet and which processing  is handled here so that we can gauge whether the code is acually needed or not.

Reply:
> Are you sure it's not removed?

Indeed, my bad, it is removed alright.

Reply:
> It would be possible to add a check for consistency here but rejecting the packet should probably be done when parsing it, not randomly during unrelated processing.

Sounds reasonable to me.